packages/rest/src/router/metadata.ts

```diff
@@ -222,14 +253,9 @@ export function operation(verb: string, path: string, spec?: OperationObject) {
       '@operation decorator can be applied to methods only',
     );
 
-    let endpoints = Object.assign(
-      {},
-      Reflector.getMetadata(ENDPOINTS_KEY, target),
-    );
-    Reflector.defineMetadata(ENDPOINTS_KEY, endpoints, target);
-
-    let endpoint: Partial<RestEndpoint> = endpoints[propertyKey];
-    if (!endpoint) {
+    const endpoints = getEndpoints(target);
```

Comment:
Should we change the tsdocs for this function to say that child classes can override REST operations?